CMakeLists.txt

```
@@ -9,7 +9,7 @@ project(ignition-utils2 VERSION 2.0.0)
 # Find ignition-cmake
 #============================================================================
 # If you get an error at this line, you need to install ignition-cmake
-find_package(ignition-cmake2 2.0.0 REQUIRED)
+find_package(ignition-cmake3 2.0.0 REQUIRED)
```

Ahh I foresee this happening with all PRs targeting the source code. It may be worth it trying to catch this case in the script.

```suggestion
find_package(ignition-cmake3 3.0.0 REQUIRED)
```

---

if it was able to detect the error and stop for manual fixing for submitting the pull request, that wouldn't be too bad, right?